Example/README.md

```diff
@@ -0,0 +1,37 @@
+# FaunaDB Driver Example Project
+
+This is a simple CRUD project we made in order to show how to use the Swift
+driver along with a iOS project.
+
+## Requirements
+
+- [CocoaPods](https://cocoapods.org/)
+
+## Running the example
+
+- Run `pod install` to setup the project
+- Using XCode 8+, open the workspace `Example.xcworkspace`
+- Edit (or duplicate) the scheme "Example" setting your key's secret at the
```

Comment:
probably break this into a few clauses for clarity:

Edit (or duplicate) the scheme "Example", and set your FaunaDB Key's secret (maybe link to our docs here?) in the environment variable `FAUNA_ROOT_KEY